porting/first-steps.rst

```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc lzop imagemagick libncurses5 rsync
 
+.. Note::
+    repo is not available in repositories of Ubuntu 20.04 or later versions. Hence it has to be installed manually.
+
+
+Installing repo 
+^^^^^^^^^^^^^^^^^
```

Comment:
trim to the length of the headline please. Also I guess this should be a level deeper, ie inside Ubuntu section, not on same level as Ubuntu and Arch sections

Reply:
Corrected as part of next commit.